README.rst

```diff
@@ -200,6 +203,49 @@ The template variables list:
 * ``${prenotazioni_folder_title}`` - Booking folder title.
 * ``${booking_requirements}`` - Related PrenotazioneType.booking_requirements field
 
+Note that the sms can be used only if you implement an own sender adapter
+Example:
+
+You just need to register a new adapter::
+
+    <adapter
+      factory = ".my_adapter.SMSSenderAdapter"
+    />
+
+And here the `send` method must bu implementend::
+
+    from logging import getLogger
+
+    from zope.component import adapter
+    from zope.component import getUtility
+    from zope.interface import Interface
+    from zope.interface import implementer
+
+    from redturtle.prenotazioni.content.prenotazione import IPrenotazione
+    from redturtle.prenotazioni.interfaces import IBookingNotificationSender
+    from redturtle.prenotazioni.interfaces import IBookingNotificatorSupervisorUtility
+    from redturtle.prenotazioni.interfaces import IPrenotazioneSMSMessage
+    from redturtle.prenotazioni.behaviors.notification_sms.adapter import BookingNotificationSender
+    logger = getLogger(__name__)
+
+
+    @implementer(IBookingNotificationSender)
+    @adapter(IPrenotazioneSMSMessage, IPrenotazione, YourAddonLayerInterface)
+    class CustomSMSSenderAdapter(BookingNotificationSender):
+        def __init__(self, message_adapter, booking, request) -> None:
```

Comment:
you don't need to reimplement the init, if you extend a class that already implement it